### DISCUSSION TITLE
Minor issues with builds since early August

### DISCUSSION THREAD

**Discussion by gitanO0** [NightscoutFoundation/xDrip#2403] [General]
Since the August  3rd release of xdrip+ I've been seeing a couple minor odd issues. I've loaded all the nightlies after August 3rd and these issues are there.  If I reload August 3rd, the issues go away. 

1) The IoB text at the top of the main screen, just under the xDrip+ title, is very flaky, sometimes it's there but most of the time that text just says insulin xxx or carbs yyy but as of August 3 release and earlier there should also always be an IoB text... assuming you actually have insulin on board.

2) I have the heart and steps functionality enabled through communications with my Fitbit and again, at or before the august 3 release, the heartbeat graph data was a nice smooth contiguous line. In the latest releases, that line is broken up and discontinuous. I've gone back and forth between builds and this seems to be a by-product of the builds, not the input data. Have there been any reports of similar issues? If so, any word on what's going on?

**Comment by Navid200** [comment]
>  If I reload August 2, the issues go away.

This is curious.  Only 2 files changed on August 3:
LibreReceiver
strings.xml

What is the data source?

**Comment by gitanO0** [comment]
G6 for the main data... but for the insulin (iob) I manually enter that as I'm still on pens.  For the heart and steps stuff, that's coming from my Fitbit.  I'll have to go back and verify 100% that it's the 2nd of August.  I could be any of the builds right after that.

**Reply by Navid200** [reply]
It helps a lot if you can identify 100%

**Comment by gitanO0** [comment]
Ok, so all is good on the 2nd of August.  August 3rd has what look to be 5 releases, a normal and 4 variants.  I haven't tried any of those yet.  Then the next release on the 18th has the two issues.  What's the scoop with the 3rd and should I try all 5?

**Reply by Navid200** [reply]
No, you shouldn't try all 5.
But, you should try the one that does not have variant in the name.

**Reply by gitanO0** [reply]
The one from the 3rd is good to go.  So it looks like something between that release and the 18th.

**Reply by Navid200** [reply]
It's great that you have identified exactly which release is responsible.  Thanks.
I am going to try to reproduce the problem.

In the meantime, would you please edit your very first post in this thread and correct the releases now that you know the problem starts on August 18 and not 3?  This helps for others who may read.  

The very first post is always very important.   Someone who is scanning all threads may only look at first posts in all threads deciding which one to focus on.

**Reply by gitanO0** [reply]
Done

**Reply by Navid200** [reply]
@gitanO0 I am quite busy with Nightscout on Google Cloud right now.  
Will you do me a favor please?  If I don't update here in a week, will you please post here so that I get a reminder?

**Reply by gitanO0** [reply]
Yes

**Reply by gitanO0** [reply]
Hello, just firing off a reminder on this.  :)

**Reply by Navid200** [reply]
Please open an issue.
In the first post, please add a link to this thread.
Please include screenshots showing the two consecutive releases behaving differently.
Please specify both releases in the post.  The consecutive releases.

Please give it a clear title.  For example, instability of insulin on board text on main screen.
You can choose.  

Thanks

**Reply by Navid200** [reply]
I thought you tested the August 3rd release and the one after, August 18.  So, two consecutive releases.
And I thought August 3rd was fine and August 18 was not.

But, I don't see this clarified in the issue you have opened.  In fact, you make no reference to the August 18 release in the issue.

It is very helpful if you have identified two consecutive releases.  If you have, please clarify it in the issue.
